Character creation and load changes

jehgo opened the pull request:
- Added correct starting locations for each race at character create
- Fixed the loading of Costumes, Quests and Mail entities; Issue #270

Rawaho commented:
Database fix I approve but the character locations shouldn't be hard coded like this.
I think as you have been made aware there is also #152 which contains an attempt at starting locations, could take some inspiration from there.

If you want to remove the character creation changes and just have the database changes I'll approve, once the start location changes are more fleshed out it can be a new PR.

jehgo commented:
Removed the hardcoded locations.
Also added an additional fix for the max character level bug I mentioned on discord, essentially just move calculation below the loading of the stats for the character so it actually has a level value to check

jehgo commented:
Closing this as I'll be opening a new PR containing the fixes in the one
